content/docs/user-guide/components/reference/terrain/terrain_surface_gradient_list.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Terrain Surface Gradient List component
+linktitle: TerrainSurfaceGradientList
+description: ' Open 3D Engine (O3DE) Terrain Surface Gradient List reference. '
+weight: 100
+---
+
+The **Terrain Surface Gradient List** component allows you to define mappings between a gradient and a surface on a terrain layer.
+You select a gradient either by dragging a entity containing a gradient component to the **GradientEntity** field, or by using the select button.
+Once a gradient is assigned, you can select the surface type that this gradient represents, by using the **Surface Tag** option.
+You can configure the dimensions and priority of the layer using the required [Terrain Layer Spawner](/docs/user-guide/components/reference/terrain/layer_spawner)
+
+## Provider ##
+
+[Terrain Gem](/docs/user-guide/gems/reference/environment/terrain)
+
+## Properties ##
+
+| Property | Description | Values | Default |
+|-|-|-|-|
+| **GradientEntity** | The gradient entity to assign to this layer. | Gradient Entity | None |
+| **Surface Tag** | Sets the surface type that this gradient repersents. | Surface Tag | Mone |
+
+
```

Comment:
As mentioned earlier, the provided link to the layer_spawner appears to be incorrect.
```suggestion

## Usage
1. Select a gradient by dragging an entity with a **Gradient** component to the **GradientEntity** field, or by using the select button.

2. Select the surface type that this gradient represents, by using the **Surface Tag** option.

3. Configure the dimensions and priority of the layer using the required [Terrain Layer Spawner](/docs/user-guide/components/reference/terrain/layer_spawner)

```

Reply:
This works now.